styles/main.css

```diff
@@ -122,30 +123,37 @@ nav ul li{
 }
 
 nav {
-    padding: 8px 2em;
-    border: 0 0 1px 0 dotted rgb(0,0,0,0.3);
-    margin: -4px 0 0 0 
+    border-bottom: 1px dotted rgb(0,0,0,0.3);
+    margin: -4px 2em 0 2em;
+    overflow: auto;

```

Comment:
This overflow declaration is the one that makes the nav element's height include the height of its floated children.